metabase/shoppy
# Do not apply gradient text in PDF/PNG exports

Avoid applying the CSS overrides for gradient text when in PDF export mode. The gradient text is a demo-specific hack and not a SDK feature.

Note that PDF exports are now **disabled** until all the visual artifact issues are resolved, so you will have to change `withDownloads={false}` to `withDownloads={true}` in order to test this.

# How to test

- Change `withDownloads={false}` to `withDownloads={true}` in `DashboardPage.tsx` in Shoppy
- Navigate to http://localhost:3004/admin/analytics/17 or any dashboard in Shoppy
- Try both the "Export as PDF" and "Dashcard > Download Results > .png" option
- The gradient background should be gone.

# Before

![image](https://github.com/user-attachments/assets/850ddf55-858f-4dd5-a756-60f59c5a360f)

![image](https://github.com/user-attachments/assets/ef14995a-d127-443f-b9b6-1478262d61e9)

# After

The gradient text attributes and font-size are no longer applied. Other visual artifact from the SDK is still there.

![CleanShot 2567-10-16 at 02 04 27@2x](https://github.com/user-attachments/assets/dfb6ad76-8268-4d63-8e71-15a69e0937a7)

![CleanShot 2567-10-16 at 02 04 53@2x](https://github.com/user-attachments/assets/572a4192-9f6d-4cb5-b95f-26c2696b0276)
